src/interfaces/swig/GaussianProcess_includes.i

```diff
@@ -1,48 +1,6 @@
 %{
-
- #include <shogun/evaluation/DifferentiableFunction.h>
  #include <shogun/machine/gp/LikelihoodModel.h>
- #include <shogun/machine/gp/ProbitLikelihood.h>
- #include <shogun/machine/gp/LogitLikelihood.h>
- #include <shogun/machine/gp/SoftMaxLikelihood.h>
- #include <shogun/machine/gp/GaussianLikelihood.h>
- #include <shogun/machine/gp/StudentsTLikelihood.h>
-
- #include <shogun/machine/gp/VariationalLikelihood.h>
- #include <shogun/machine/gp/VariationalGaussianLikelihood.h>
- #include <shogun/machine/gp/NumericalVGLikelihood.h>
- #include <shogun/machine/gp/DualVariationalGaussianLikelihood.h>
- #include <shogun/machine/gp/LogitVGLikelihood.h>
- #include <shogun/machine/gp/LogitVGPiecewiseBoundLikelihood.h>
- #include <shogun/machine/gp/LogitDVGLikelihood.h>
- #include <shogun/machine/gp/ProbitVGLikelihood.h>
- #include <shogun/machine/gp/StudentsTVGLikelihood.h>
-
  #include <shogun/machine/gp/MeanFunction.h>
- #include <shogun/machine/gp/ZeroMean.h>
- #include <shogun/machine/gp/ConstMean.h>
-
  #include <shogun/machine/gp/Inference.h>
- #include <shogun/machine/gp/LaplaceInference.h>
- #include <shogun/machine/gp/SparseInference.h>
- #include <shogun/machine/gp/SingleFITCInference.h>
- #include <shogun/machine/gp/SingleLaplaceInferenceMethod.h>
- #include <shogun/machine/gp/SingleSparseInference.h>
- #include <shogun/machine/gp/MultiLaplaceInferenceMethod.h>
- #include <shogun/machine/gp/ExactInferenceMethod.h>
- #include <shogun/machine/gp/FITCInferenceMethod.h>
- #include <shogun/machine/gp/VarDTCInferenceMethod.h>
- #include <shogun/machine/gp/SingleFITCLaplaceInferenceMethod.h>
- #include <shogun/machine/gp/EPInferenceMethod.h>
-
- #include <shogun/machine/gp/KLInference.h>
- #include <shogun/machine/gp/KLLowerTriangularInference.h>
- #include <shogun/machine/gp/KLCovarianceInferenceMethod.h>
- #include <shogun/machine/gp/KLDiagonalInferenceMethod.h>
- #include <shogun/machine/gp/KLCholeskyInferenceMethod.h>
- #include <shogun/machine/gp/KLDualInferenceMethod.h>
-
- #include <shogun/machine/GaussianProcessMachine.h>
- #include <shogun/classifier/GaussianProcessClassification.h>
- #include <shogun/regression/GaussianProcessRegression.h>
+ #include <shogun/machine/GaussianProcess.h>
```

Comment:
same here, would be better to delete the file and move it all to the Machine file

Reply:
i have tried many times that move those to the Machine file, but i still haven't succeeded, the swig gives me very weird errors.

Reply:
You get errors from moving the header files?

Reply:
Does it matter that the order of the swig file?

Reply:
I would assume you don’t get any errors for the header files in the _includes.i files. These are just literal cpp code snippets pasted on the swig generated cpp

Reply:
i guess those errors from moving the `GaussianProcess.i` to `Machine.i`

Reply:
that's very likely. Order matters in swig so you need to try and figure out what the pattern of the order of the files is and maybe read the docs about this. What is the error you get btw?

Reply:
error messages are something like `Parameters are not be defined in this scope`